packages/pigment-css-core/src/keyframes.js

```diff
@@ -0,0 +1,10 @@
+import { generateErrorMessage } from './utils';
+
+/**
+ * `__wyw_dynamic_import` is a special global var that is set during the evaluation phase by wyw.
+ * So during eval phase, it can happen that some code is calling the runtime function.
+ * We do not want to throw error in that case as we want the evaluation to happen.
+ */
+export default function keyframes() {
```

Comment:
Is there a reason not to author those files in typescript? Aren't they currently being built by `tsup`?

Reply:
Wouldn't that work if the function returns a `string`?